pkg/webui/console/views/application-api-keys-list/index.js

```diff
@@ -20,21 +20,38 @@ import IntlHelmet from '../../../lib/components/intl-helmet'
 import ApiKeysTable from '../../containers/api-keys-table'
 import { getApplicationApiKeysList } from '../../store/actions/applications'
 import sharedMessages from '../../../lib/shared-messages'
+import PropTypes from '../../../lib/prop-types'
+
+import {
+  selectApplicationApiKeys,
+  selectApplicationApiKeysTotalCount,
+  selectApplicationApiKeysFetching,
+} from '../../store/selectors/applications'
 
 import PAGE_SIZES from '../../constants/page-sizes'
 
-@bind
 export default class ApplicationApiKeys extends React.Component {
+  static propTypes = {
+    match: PropTypes.match.isRequired,
+  }
+
   constructor(props) {
     super(props)
 
     const { appId } = props.match.params
     this.getApplicationsApiKeysList = filters => getApplicationApiKeysList(appId, filters)
   }
 
-  baseDataSelector({ apiKeys }) {
+  @bind
+  baseDataSelector(state) {
     const { appId } = this.props.match.params
```

Comment:
It might make sense to take the opportunity here to replace this with `selectedSelectedApplicationId()` selector. Likewise for the other entities.